Makefile

```diff
@@ -4,13 +4,11 @@ PREFIX ?= /
 
 .PHONY: install
 install:
-	install -D -m0644 src/overdue.awk $(PREFIX)/usr/share/overdue/overdue.awk
 	install -D -m0644 src/overdue-libraries-upgrade.hook \
 		$(PREFIX)/usr/share/libalpm/hooks/overdue-libraries-upgrade.hook
-	install -D -m0755 src/overdue.sh  $(PREFIX)/usr/bin/overdue
```

Comment:
I like piping in library files, whether the file sources come from `find` or `lsof`. Maybe the quickest approach is to get deleted handles from `lsof -d DEL`, munge them to just get individual file names and pass that into the script. Then it's just piping in a subset of library files, not the entirety of `/usr/lib`.